user-manual/transfer/dataverse.rst

```diff
@@ -0,0 +1,118 @@
+.. _dataverse-transfers:
+
+=====================
+Dataverse Integration
+=====================
+
+.. _dataverse-overview:
+
+Overview
+--------
+
+`Dataverse`_ is an open source repository for research data. Archivematica can
+be configured to use a Dataverse Repository as a Transfer Source Location. 
+Dataverse Transfer Source Locations can be configured to display all available 
+datasets or a subset using query parameters.
+
+Datasets are retrieved directly using the Dataverse API and processed like 
+standard transfers, with some additional processing as described below.
+
+Dataverse integration is supported by Archivematica version 1.8 and above, and
+has been developed and tested using Dataverse version 4.8.6
+
+*On this page*
+
+* :ref:`Selecting Datasets for preservation <selecting-datasets>`
+* :ref:`Processing Dataverse datasets <processing-datasets>`
+* :ref:`Dataverse METS file <dataverse-mets>`
+* :ref:`AIP METS file <dataverse-aip-mets>`
+* :ref:`Configuration <dataverse-configuration>`
+
+.. _selecting-datasets:
+
+Selecting Datasets for preservation
+-----------------------------------
+
+When a Dataverse Transfer Source Location is selected in the Transfer Tab of 
```

Comment:
I'm not sure about the capitalization of 'Transfer Tab'

Reply:
good spot. It looks like in the rest of the docs 'Transfer tab' is used so am changing to that.